src/lib/y2network/interface_config_builder.rb

```diff
@@ -309,7 +308,7 @@ def aliases=(value)
 
     # gets interface name that will be assigned by udev
     def udev_name
-      # cannot cache as EditNicName dialog can change it
+      # cannot cache as it can be changed
```

Comment:
can we replace that value now? by new backend instead of old one?

Reply:
Yes, you are right. I will have a look and check whether we can drop this old line of code.